plugins/woocommerce-beta-tester/.wp-env.json

```diff
@@ -1,7 +1,4 @@
 {
 	"phpVersion": "7.4",
-	"plugins": [ 
-        ".",
-        "https://downloads.wordpress.org/plugin/woocommerce.zip"
-     ]
+	"plugins": [ "https://downloads.wordpress.org/plugin/woocommerce.zip", "." ]
```

Comment:
The order of this array defines the order the plugins are installed. WooCommerce needs to be installed before WooCommerce Beta Tester.

Reply:
This looks good to me!